### DISCUSSION TITLE
FreeTDS library filename normalizing

### DISCUSSION THREAD

**Discussion by taozuhong** [FreeTDS/freetds#518] [General]
![image](https://github.com/FreeTDS/freetds/assets/13451009/0d7e8571-fe73-4964-9ca9-3182ff0c8799)
![image](https://github.com/FreeTDS/freetds/assets/13451009/f369d35c-4069-4504-940d-ca757bfaba85)
![image](https://github.com/FreeTDS/freetds/assets/13451009/51d4ec8a-df05-44c9-a61f-17de7075e793)

Maybe we could normalize the library filename as follow:
CT-Library:
```
tdsct.pc
libtdsct.dll
libtdsct.dll.a
libtdsct_static.a
```
DB-Library: 
```
tdsdb.pc
libtdsdb.dll
libtdsdb.dll.a
libtdsdb_static.a
```
ODBC library:
```
libtdsodbc.dll
libtdsodbc.dll.a
libtdsodbc_static.a
```

**Comment by freddy77** [comment]
There should be no `libtdsodbc_static.a`. Beside that Windows libraries IMHO should match Unix one and I would keep ABI, otherwise I'll have to wait like 10 years (no joking!) before Debian pick up our changes.

**Reply by taozuhong** [reply]
Ah, it could make a symbolic link to keep ABI as you wish.
